src/assets/scripts/aircraft/AircraftInstanceModel.js

```diff
@@ -1699,6 +1699,27 @@ export default class Aircraft {
     }
 
     /**
+     * Checks if the aircraft is inside the airspace of a specified airport
+     *
+     * @for AircraftInstanceModel
+     * @method isInsideAirspace
+     * @param  {airport} airport the airport whose airspace we are checking
+     * @return {Boolean}
+     * @private
+     */
+    isInsideAirspace(airport) {
```

Comment:
totally cool.  I just wanted to call that out in case you ever need it in the future 😉